tests/lib/rules/html-checker/tests.ts

```diff
@@ -185,5 +232,9 @@ ruleRunner.testRule(ruleName, testsForValidatorConfig, {
     ruleOptions: { validator: configValidator },
     serial: true
 });
+ruleRunner.testRule(ruleName, testsForDetailsConfig, {
+    ruleOptions: { details: true },
+    serial: true
```

Comment:
Do we need it to be `serial`?

Reply:
@molant Yes. I'm not sure if it's because of the use of `mock`, running tests in parallel doesn't work.